.github/workflows/ci.yml

```diff
@@ -13,10 +13,71 @@ on:
       - release/**
 
 jobs:
+# add get-diff job 
+  get-diff:
+    runs-on: ubuntu-latest
+    # Map a step output to a job output
+    outputs:
+      output-contract: ${{ steps.contract.outputs.run }}
+      output-certora-contract: ${{ steps.certora-contract.outputs.run }}
+      output-scripts: ${{ steps.scripts.outputs.run }}
+      output-agents: ${{ steps.agents.outputs.run }}
```

Comment:
shouldn't prefix the output name with `output`, it's redundant.
Compare these two, which looks better?
* `steps.get-diff.outputs.output-contract`, OR
* `steps.get-diff.outputs.contract`
??

Reply:
Done